strings/Longest_common_prefix.cpp

```diff
@@ -0,0 +1,51 @@
+//  A C++ Program to find the longest common prefix
+#include<bits/stdc++.h>
+using namespace std;
+
```

Comment:
As the functions do only read strings it would make sense to qualify the string arguments as const.

Reply:
i'm updating for the same